cmd/kube-apiserver/app/server.go

```diff
@@ -210,9 +212,16 @@ func Run(s *options.APIServer) error {
 				installSSH = instances.AddSSHKeyToAllInstances
 			}
 		}
-
+		if ports.KubeletReadOnlyPort <= 0 {
```

Comment:
The intent was to make sure that port was enabled on your kubelet (which it doesn't actually do). I think what I want is `s.KubeletConfig.Port`. Same net result (doesn't actually confirm that port is listening), but at least it's actually a parameter.